modules/claims_api/config/schemas/v2/2122a.json

```diff
@@ -79,7 +78,7 @@
               "example": "USA"
             },
             "zipCode": {
-              "description": "Zipcode (First 5 digits) of the address.",
+              "description": "Zip code (First 5 digits) of the address.Required if 'country' is 'US'.",
```

Comment:
Same comment as above (missing space between `address.Required`). Also, we refer to `country` and `countryCode` interchangeably throughout this PR. Should we stick to one?